DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php

```diff
@@ -375,6 +375,10 @@ public function fixTemplates(ContainerBuilder $container, Definition $definition
             'ajax' => 'SonataAdminBundle::ajax_layout.html.twig',
             'dashboard' => 'SonataAdminBundle:Core:dashboard.html.twig',
             'list' => 'SonataAdminBundle:CRUD:list.html.twig',
+            'list__action' => 'SonataAdminBundle:CRUD:list__action.html.twig',
```

Comment:
Do we really need this override? You can replace this with the normal twig template overrides using `ROOT/templates/SonataAdminBundle/...`?